zerver/tests/test_upload.py

```diff
@@ -689,7 +691,10 @@ def check_xsend_links(name: str, name_str_for_test: str,
                 _get_sendfile.clear()
                 test_upload_dir = os.path.split(settings.LOCAL_UPLOADS_DIR)[1]
                 self.assertEqual(response['X-Accel-Redirect'],
-                                 '/serve_uploads/../../'  + test_upload_dir +
+                                 '/serve_uploads/../../' +
+                                 get_dev_uuid_var_path().split('/', 5)[-1] +
```

Comment:
This line is completely incomprehensible.  Do you just want `os.path.basename()`?  

Also, can't this just use `LOCAL_UPLOADS_DIR` as an input?  That already has `get_dev_uuid_var_path` in it...

Reply:
Yes, I agree 😅. I was trying to see how the paths matched without much regard to the manner in which I was doing it. 

Sorry this slipped my mind. I'll certainly clean it up, thank you for the advice.

Reply:
The change I made should now be self-explanatory with what I was trying to do.